source/simulator/core.cc

```diff
@@ -130,7 +130,9 @@ namespace aspect
             this_mpi_process(mpi_communicator)
             == 0)),
 
-    computing_timer (pcout, TimerOutput::never,
+    computing_timer (mpi_communicator,
+                     pcout,
```

Comment:
Uh oh, does that mean that we had previously implicitly used `MPI_COMM_WORLD`? That would be a nice fat bug you squashed here!